components/BatchProcessor.tsx

```
@@ -607,22 +675,20 @@ export default function BatchProcessor({ doorContexts, onComplete, modelSource }
             <div className="image-modal-header">
               <h3>{modalImage.doorId} - {modalImage.view.charAt(0).toUpperCase() + modalImage.view.slice(1)} View</h3>
               <button className="close-button" onClick={closeModal} aria-label="Close">
-                ×
+                <X size={18} />
               </button>
             </div>
             <div className="image-modal-body">
-              <div 
+              <div
                 className="image-container"
                 dangerouslySetInnerHTML={{ __html: modalImage.svg }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)